src/p11.rs

```diff
@@ -138,7 +142,7 @@ impl SymKey {
     ///
     /// # Errors
     /// Internal errors in case of failures in NSS.
-    pub fn as_bytes(&self) -> Res<&[u8]> {
```

Comment:
This looks suspicious, and I believe this breaks existing code like authenticator_rs

Reply:
fixed by re-adding right below